website/content/docs/k8s/annotations-and-labels.mdx

```diff
@@ -50,6 +50,10 @@ The following Kubernetes resource annotations could be used on a pod to control
   list of additional user IDs to exclude from traffic redirection when running in transparent proxy
   mode.
 
+- `consul.hashicorp.com/use-proxy-health-check` - It this is set to "true" or truthy values, it configures
```

Comment:
```suggestion
- `consul.hashicorp.com/use-proxy-health-check` - It this is set to "true", it configures
```

Reply:
I believe just or true or false are allowed?

Reply:
Truth should also be in back ticks and not quotation marks per our style guide.